docker-compose.yml

```diff
@@ -1,5 +1,35 @@
 version: '3'
 services:
+  ssm-postgres:
+    image: postgres
+    restart: unless-stopped
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_PASSWORD: postgres
+
+  ssm-keycloak:
+    image: keycloak
+    restart: unless-stopped
+    depends_on:
+      - ssm-postgres
+    ports:
+      - "8080:8080"
+    environment:
+      FEDERATE_LDAP: "true"
+      GOOGLE_CLIENT_ID: 308694469630-7l6odifs3dc88arbh5cia6ud7v4h2suo.apps.googleusercontent.com
+      GOOGLE_IDP: "true"
+      GOOGLE_SECRET: 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
+      HOSTNAME: ssm-keycloak
+      KC_DB: postgres
+      KC_DB_PASSWORD: postgres
+      KC_DB_URL: jdbc:postgresql://ssm-postgres:5432/
+      KC_DB_USERNAME: postgres
+      KEYCLOAK_ADMIN: admin
+      KEYCLOAK_ADMIN_PASSWORD: admin
+      LDAP_URL: ldaps://ldapu.ornl.gov
+      USERSDN: "dc=ornl,dc=gov"
```

Comment:
Let's also take out this ldap server, we need to try and pull out ORNL specific stuff